### PR TITLE
cloud: Update cluster creation steps

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by easonn7** [comment]
LGTM

File: tidb-cloud/release-notes-2022.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.
 
+## June 15, 2022
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:11]:
```suggestion
## June 16, 2022
```

File: tidb-cloud/release-notes-2022.md [hunk 2]
```diff
@@ -8,6 +8,14 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.
 
+## June 16, 2022
+
+* Simplify the cluster creation process:
```
Comment (qiancai) [inline tidb-cloud/release-notes-2022.md:13]:
```suggestion
* Simplify the cluster creation process
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 3]
```diff
@@ -57,56 +59,50 @@ Your TiDB Cloud cluster will be created in approximately 5 to 15 minutes.
...
 
     The overview page of your newly created cluster is displayed.
 
 2. Click **Connect**. The **Connect to TiDB** dialog box is displayed.
 
-3. In **Step 1: Create traffic filter**, click **Add Your Current IP Address**, and then click **Create Filter**.
+3. In the **Connect to TiDB** dialog box, copy the connection string in **Step 2: Connect with a SQL client** . For example:
 
-    The purpose of this step is to set up your traffic filter, which makes sure that the cluster accepts connections only from trusted IP addresses.
+    `mysql --connect-timeout 15 -u root -h tidb.4aa56280.2c524d28.ap-northeast-1.prod.aws.tidbcloud.com -P 4000 -p`
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-quickstart.md:97]:
````suggestion
    ```shell
    mysql --connect-timeout 15 -u root -h tidb.4aa56280.2c524d28.ap-northeast-1.prod.aws.tidbcloud.com -P 4000 -p
    ```
````

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Pull request merged**